types/microsoft-graph/index.d.ts

```diff
@@ -26567,8 +26792,8 @@ export interface Process {
 }
 export interface RegistryKeyState {
     /**
-     * A Windows registry hive : HKEY_CURRENT_CONFIG HKEY_CURRENT_USER HKEY_LOCAL_MACHINE/SAM HKEY_LOCAL_MACHINE/Security
-     * HKEY_LOCAL_MACHINE/Software HKEY_LOCAL_MACHINE/System HKEY_USERS/.Default. Possible values are: unknown, currentConfig,
+     * A Windows registry hive : HKEYCURRENTCONFIG HKEYCURRENTUSER HKEYLOCALMACHINE/SAM HKEYLOCALMACHINE/Security
```

Comment:
Ok, something is definitely wrong, these reg keys should have underscores in them.

Reply:
Thanks for capturing this, I'll investigate the issue